index.js

```diff
@@ -1,20 +1,27 @@
-const gameStartElement = document.getElementById('start-game')
+const gameStartElement = document.getElementById("start-game");
 
-gameStartElement.addEventListener('click', (event) => {
-    alert('Game is under construction')
-})
+gameStartElement.addEventListener("click", event => {
+  alert("Game is under construction");
+});
+
+// Create gameboard
+
+const canvas = document.getElementById('gameboard');
+const ctx = canvas.getContext('2d');
+
+ctx.fillStyle = 'lightblue';
+ctx.fillRect(10, 10, 500, 500);
 
 document.addEventListener("keydown", event => {
-    if (event.isComposing || event.keyCode === 229) {
-        return;
-    }
-    else if (event.keyCode === 40) {
-        console.log('KeyDown is pressed')
-    } else if (event.keyCode === 38) {
-        console.log('KeyUp is pressed')
-    } else if (event.keyCode === 37) {
-        console.log('KeyLeft is pressed')
-    } else if (event.keyCode === 39) {
-        console.log('KeyRight is pressed')
-    }
-})
+  if (event.isComposing || event.keyCode === 229) {
+    return;
+  } else if (event.keyCode === 40) {
```

Comment:
You don't have to use keyCodes.  Just have a quick look at these documents:

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values